### ISSUE TITLE
I ran the .exe in final_finals

### ISSUE THREAD

**Issue by edwardlennonjr** [OOW/SWMMout2csv#5]
![image](https://user-images.githubusercontent.com/8989544/30293972-69378a0a-9709-11e7-9b03-a71d10a2bfef.png)

**Comment by edwardlennonjr** [comment]
It just needs to be updated. Also isn't there a setting that adds all required windows files into the exe file. I can't recall what it's called.

**Comment by shanninio-jing** [comment]
I guess it's probably because the exe was generated with Debug Build, I used debug build as well just to match the master branch. I think we won't have this issue if the exe is generated with Release Build. We can give it a try. I will add a new branch to include the exe generated with Release Build

**Comment by edwardlennonjr** [comment]
![image](https://user-images.githubusercontent.com/8989544/30294587-97dc3ee4-970b-11e7-8f19-56be43de1333.png)
I compiled in release mode with these setting. It works for me now. Need to test on a unsuspecting computer.

**Comment by edwardlennonjr** [comment]
https://stackoverflow.com/questions/15297270/problems-with-running-exe-file-built-with-visual-studio-on-another-computer

**Comment by shanninio-jing** [comment]
the master branch works fine on Chris's computer

**Comment by shanninio-jing** [comment]
I will test it again on my laptop tonight to see if it works

**Comment by shanninio-jing** [comment]
I seems like it could work if the exe is generated with the Release Build without changing any default settings in Visual Studio. I tested with the release-test branch. The exe in release-branch was generated with Release Build(everything else is in default). And I deleted the line 136 you just added in  SWMMout2csv.vcxproj. It worked fine on Chris's computer. Also, I will test it again tonight.
